### PR TITLE
Fix confusing wording in the sign in banner

### PR DESCRIPTION
**Pull request by ClearlyClaire**
The sign in banner currently has the following text:

> Sign in to follow profiles or hashtags, favourite, share and reply to posts, or interact from your account on a different server.

While the intent is to make clear that the interactions can be performed from another server, this ends up misleading people into thinking they can log in with the credentials from a remote server. Therefore, I propose the following rewording:

> Sign in to follow profiles or hashtags, favourite, share and reply to posts (you can also interact from your account on a different server).

### CODE REVIEW

**Comment by trwnh** [comment]
i'd reframe the sentence so the intent comes first:

> Before you can follow, favourite, boost, or reply, you'll need to sign in. You can either sign into an account on $SERVER_NAME, or if you have an account on a different server, you can interact from there by pasting a URL into your search box.

**Comment by ClearlyClaire** [comment]
> i'd reframe the sentence so the intent comes first:
> 
> > Before you can follow, favourite, boost, or reply, you'll need an account.
> > You can either sign into or create an account on $SERVER_NAME, or if you have an account on a different server, you can interact from there by pasting a URL into your search box.

I don't think that's the place for the whole URL thing.

**Comment by trwnh** [comment]
ok then just leave out the bit about the url

> Before you can follow, favourite, boost, or reply, you'll need an account. You can either sign into or create an account on $SERVER_NAME, or if you have an account on a different server, you can interact from there.

**Comment by Plastikmensch** [comment]
> ok then just leave out the bit about the url
> 
> > Before you can follow, favourite, boost, or reply, you'll need an account. You can either sign into or create an account on $SERVER_NAME, or if you have an account on a different server, you can interact from there.

It's a bit long, but I like the idea of specifying the domain in the text.

I'm not sure if specifying interactions is really needed though, as the modal popping up when you try to interact gives all the information needed, so 

> Sign in or create an account on {domain}

is all that's really needed imo.

Also, related issue: #22403

**Comment by trwnh** [comment]
how about something like 

> With an account on $SERVER_NAME, you can participate in the local community as well as talk to people on other servers.

**Comment by Plastikmensch** [comment]
> how about something like
> 
> > With an account on $SERVER_NAME, you can participate in the local community as well as talk to people on other servers.

Isn't that what pretty much the old text was?
I'm not sure that wouldn't create a new problem, making people think they \*need\* an account on the domain to do that.

**Comment by trwnh** [comment]
> With an account on $SERVER_NAME, you can participate in the local community as well as talk to people on other servers. You can also sign up elsewhere on [Mastodon servers](https://joinmastodon.org/servers) and talk to anyone across the whole network.

**Comment by Plastikmensch** [comment]
> > With an account on $SERVER_NAME, you can participate in the local community as well as talk to people on other servers. You can also sign up elsewhere on [Mastodon servers](https://joinmastodon.org/servers) and talk to anyone across the whole network.

I would reword the second sentence and remove the link, as the page is already plastered with links to the website and the server list is only a very tiny fraction of instances, but otherwise I'm fine with it.

**Comment by samuk** [comment]
> Welcome to {domain} community server. You'll be able to talk to anyone across the whole network.

Would give 100% reach according to https://app.readable.com/text/?demo

**Comment by Plastikmensch** [comment]
> > Welcome to {domain} community server. You'll be able to talk to anyone across the whole network.
> 
> Would give a 'grade A' 100% reach according to https://app.readable.com/text/?demo

That's too simplified and makes little sense.

**Comment by samuk** [comment]
>Welcome to {domain} community server. With an account here you'll be able to talk to anyone across the whole network.

I guess you could add the 'with an account here part'. I'm not really attached to any particular wording, but I would like it to readable by most people.  Using Flesch- Klincaid/ Gunning Fog score seems a reasonable proxy for that.

I don't personally see much merit in drawing a distinction between the local and the network if the main point is that there isn't a big distinction. Better IMHO to just communicate what an account enables at a network level.

Could you have a play with https://app.readable.com/text/?demo and come up with a form or words that improves the readability above the current grade C score?

**Comment by ClearlyClaire** [comment]
> Thinking more about it, maybe:
> 
> > With an account on {domain}, you can participate in the local community as well as talk to anyone across the whole network.
> 
> Maybe replace "talk to" with "interact with"

Oh, I understood the intention behind the original text as “you can also interact from another server” rather than “you can also interact with other servers from this server”. Both convey that the servers are connected, though.

**Review by Gargron — APPROVED** [review]

**Comment by adamlui** [comment]
The first sentence of the new commit doesn't make sense:

> Sign in to follow profiles or hashtags, favourite, share and reply to posts. 

Structurally, the commas + "and" suggest a list of 4 possible things: 1. follow profiles or hashtags 2. favourite 3. share 4. and reply to posts , but #2 and #3 are missing objects, so they belong with #4 as a single independent clause, which is only grammatically correct when the two clauses are joined exactly where they connect, by a single conjunction:

> [Sign in to follow profiles or hashtags] and [favorite/share/reply to posts]

This is actually what makes the original sentence clearer, as it groups the fav/share/reply into a single clause by using a list of 3 items (where no conjunctions are grammatically required except preceding final elements of lists):

> Sign in to [1: follow profiles or hashtags], [2: favourite, share and reply to posts], or [3: interact from your account on a different server.]

So by splitting it into two sentences, the comma between #1 and #2 needs to be replaced with a conjunction, or you can keep all 3 in a single sentence and use semi-colons to remain grammatical/readable (but current commit does neither so becomes less so)

**Some readable solutions**

This one maintains original structure:

> Sign in to follow profiles/hashtags, favourite/share/reply to posts, or interact with other servers.' 

This one uses structure from new commit:

> Sign in to follow profiles/hashtags and favourite/share/reply to posts. You can also interact from your account on a different server.

This one captures notable functions succinctly (important to good copywriting):

> Sign in to follow profiles & hashtags, engage with new & trending posts, or even interact with other servers!
